### PR TITLE
GH-80: Update `contributing.md`

### PR DESCRIPTION
**Pull request by Fokko**
Bring it up to date with the new GitHub workflow.

As a follow up for: https://lists.apache.org/thread/x7l8y8fo4q2f0fyqjgdw2z7341hj6j71

### CODE REVIEW

File: content/en/docs/Contribution Guidelines/contributing.md [hunk 1]
```diff
@@ -9,97 +9,59 @@ description: >
 Pull Requests
 -------------
 
-We prefer to receive contributions in the form of GitHub pull requests. Please send pull requests against the [github.com/apache/parquet-mr](https://github.com/apache/parquet-mr) repository. If you’ve previously forked Parquet from its old location, you will need to add a remote or update your origin remote to [https://github.com/apache/parquet-mr.git](https://github.com/apache/parquet-mr.git) Here are a few tips to get your contribution in:
+We prefer to receive contributions in the form of GitHub pull requests. Please send pull requests against the [github.com/apache/parquet-java](https://github.com/apache/parquet-java) repository. If you’ve previously forked Parquet from its old location, you will need to add a remote or update your origin remote to [https://github.com/apache/parquet-java.git](https://github.com/apache/parquet-java.git). Here are a few tips to get your contribution in:
```
Comment (wgtmac) [inline content/en/docs/Contribution Guidelines/contributing.md:12]: Should we change the title to `Contributing to Parquet Java`? Or add a section about Parquet Format?

File: content/en/docs/Contribution Guidelines/contributing.md [hunk 2]
```diff
@@ -9,97 +9,59 @@ description: >
...
-
-    dev/merge_parquet_pr.py
-
-
-example output:
-
-    Which pull request would you like to merge? (e.g. 34):
+### Merging a Pull Request
 
+Merging a pull request requires being a committer on the project.
```
Comment (julienledem) [inline content/en/docs/Contribution Guidelines/contributing.md]: I would add "and approval of the PR by a committer who is not the author"

File: content/en/docs/Contribution Guidelines/contributing.md [hunk 3]
```diff
@@ -9,97 +9,59 @@ description: >
...
-    HEAD is now at b767ac4... Update README.md
-    Deleting local branch PR_TOOL_MERGE_PR_X
-    Deleting local branch PR_TOOL_MERGE_PR_X_MASTER
-    Pull request #X merged!
-    Merge hash: 485658a5
+Make sure that the remote is set up correctly:
 
-    Would you like to pick 485658a5 into another branch? (y/n):
+```sh
+git remote add github-apache git@github.com:apache/parquet-mr.git
```
Comment (julienledem) [inline content/en/docs/Contribution Guidelines/contributing.md]:
```suggestion
git remote add github-apache git@github.com:apache/parquet-java.git
```

**Review by julienledem — APPROVED** [review]
I made some small comments
Thank you for doing this

File: content/en/docs/Contribution Guidelines/contributing.md (hunk 1 above)
Comment (Fokko) [reply content/en/docs/Contribution Guidelines/contributing.md:12]: Yes, I think so. Thanks!

File: content/en/docs/Contribution Guidelines/contributing.md (hunk 2 above)
Comment (Fokko) [reply content/en/docs/Contribution Guidelines/contributing.md]: Since we're already doing that, I think it is good to formalize that as well 👍 Thanks

**Review by wgtmac — APPROVED** [review]

**Pull request merged**